### PR TITLE
export refactor

### PR DESCRIPTION
**Pull request by braxex**
## Description

Closes [MDCT-2193](https://qmacbis.atlassian.net/browse/MDCT-2193) and [MDCT-2195](https://qmacbis.atlassian.net/browse/MDCT-2195). 

Addressing a couple of tech debt items, increased code coverage, and implemented feedback from R&D. 

### How to test

- Generate an MCPAR report
- Export as PDF and take a look at it!
- Ensure that all required fields are rendered and any answered fields should populate

### Changed Dependencies

N/A

## Code author checklist
- [ ] I have performed a self-review of my code
- [ ] I have added [thorough](https://bit.ly/3zPrxuZ) tests
- [ ] I have added analytics, if necessary
- [ ] I have updated the documentation, if necessary

## Reviewer checklist (two different people)
- [ ] I have done the deep review and verified the items in the above checklist are g2g
- [ ] I have done the lgtm review

### CODE REVIEW

File: services/ui-src/src/components/export/ExportedReportFieldRow.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ const renderDataCell = (
   if (pageType === "drawer") {
     const entityData = reportData[entityType!];
     return entityData?.map((entity: EntityShape) => {
-      const fieldData = entityData[field.id];
+      const fieldData = entityData[0][field.id];
```
Comment (braxex) [inline services/ui-src/src/components/export/ExportedReportFieldRow.tsx:20]: Wait why is this necessary? I might just not be thinking it through clearly.

**Comment by britt-mo** [comment]
Screen Shot 2023-01-25 at 4 41 40 PM
Screen Shot 2023-01-25 at 4 43 30 PM

Should D1.II.3's selection header show up as 'N/A' in the print preview?

**Review by britt-mo — COMMENTED** [review]
Looks really good, just a question, left screenshots.

**Comment by braxex** [comment]
> Looks really good, just a question, left screenshots.

Yep, that's per design feedback. Good question. Gonna close out the conversation. 👍

File: services/ui-src/src/components/tables/Table.tsx [hunk 2]
```diff
@@ -100,12 +94,9 @@ const sx = {
...
     fontWeight: "normal",
-    color: "black",
-    ".shrink &": {
-      padding: "0.375rem 0rem",
-    },
     ".mobile &": {
       fontSize: "xs",
     },
   },
+  ".two-column &": {}, // TODO: add additional styling for two-column dynamic field tables if needed
```
Comment (tblackwe) [inline services/ui-src/src/components/tables/Table.tsx:101]: is this TODO resolved?

**Review by tblackwe — APPROVED** [review]

**Review by britt-mo — APPROVED** [review]

**Pull request merged**